src/antsibull_nox/config.py

```diff
@@ -78,6 +78,7 @@ class SessionLint(_BaseModel):
 
     default: bool = True
     extra_code_files: list[str] = []
+    ruff_config: t.Optional[p.FilePath] = None
```

Comment:
I think a global ruff_package option that behaves that has the same overridable behavior as the global ruff_config also makes sense, but other than that, this looks good to me. Thanks!

Reply:
Implemented in 7b530b94810abe4dcc4672e7837b175ca07719b4.